db/migrate/20180514234655_add_scheme_to_searchgov_domain.rb

```diff
@@ -0,0 +1,5 @@
+class AddSchemeToSearchgovDomain < ActiveRecord::Migration
+  def change
+    add_column :searchgov_domains, :scheme, :string, default: 'http', null: false, length: 5
```

Comment:
Oh man - I was actually trying to be *less* anal-retentive than usual about my migration testing, and thought I was covered by my shoulda spec (and for once I skipped the "run a failing test" step). And of course, TIL, `shoulda` doesn't complain if the options you pass to `with` are bogus. Note to self: go back to being anal-retentive.

Reply:
And yes, all your grokking is correct. I should note that in my testing, a `SearchgovDomainIndexerJob` with a blocked domain failed as desired:
```
Worker
Moths-MBP.hsd1.wa.comcast.net:55552 on SEARCHGOV at about 3 hours ago Retry or Remove Class
 SearchgovDomainIndexerJob (via ActiveJob)
Arguments
---
job_class: SearchgovDomainIndexerJob
job_id: 8da62d7e-b548-4d80-9c18-b07bb298d0c4
queue_name: searchgov
arguments:
- _aj_globalid: gid://usasearch/SearchgovDomain/4
- 1
locale: en
Exception
SearchgovUrl::DomainError
Error
SSL_connect returned=1 errno=0 state=error: certificate verify failed
```
However, I think we want any such errors to be reported to Airbrake, because we'll want to know if a domain goes down or starts blocking us. Do you know how we can keep Airbrake from swallowing those failures?